src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,11 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+
 public class CinemaHallRequestDto {
+    @NotNull
     private int capacity;
```

Comment:
Use Integer instead of int for the capacity field, because the @NotNull constraint won't work with primitive types.